valhalla/sif/dynamiccost.h

```diff
@@ -878,6 +879,9 @@ class DynamicCost {
     alley_penalty_ = costing_options.alley_penalty();
     destination_only_penalty_ = costing_options.destination_only_penalty();
     maneuver_penalty_ = costing_options.maneuver_penalty();
+    // TODO: Get this from the costing model
```

Comment:
adding this is quite straightforward:

1.  just add the float to the protobuf costing options defintion
2. set it by parsing the value here: https://github.com/valhalla/valhalla/blob/master/src/sif/dynamiccost.cc#L275, defaulting to your value of `9.f` if not specified
3. then make sure to initialize it in the constructor: https://github.com/valhalla/valhalla/blob/master/src/sif/dynamiccost.cc#L82

Reply:
other than ^^ this pr looks good!

Reply:
Done. Its now a costing option in the request. Value ranges from 1. to 10, with 1 meaning don't apply an additional penalty to closed edges. I've also limited the max value to 10 (defaults to 9) so that we don't extend the search for too long from one end.

CC: @kevinkreiser @danpat